packages/design-system/src/styles/components/_Button.scss

```diff
@@ -13,40 +13,46 @@
   font-weight: var(--fontWeight, $button__font-weight);
   line-height: 1.3;
   text-align: center;
-  text-decoration-color: transparent;
   display: inline-block;
   padding: var(--space, $spacer-1 $spacer-3);
 
+  &:is(a) {
+    text-decoration: none;
+
+    &:focus {
+      background-color: var(--backgroundColor, $button__background-color);
+      text-decoration: none;
+    }
+
+    &.ds-c-button--disabled {
+      pointer-events: none; // Needed to render disabled links correctly
```

Comment:
This change made me happy. 

We were previously applying this rule to all `disabled` buttons just to prevent accidental interaction with anchor buttons. By moving this style to a more specific rule, I was able to apply a `not-allowed` cursor to our disabled button buttons, making for a more user-friendly experience.